### DISCUSSION TITLE
Option to hide default status tracking bars - Proposal

### DISCUSSION THREAD

**Discussion by ghost** [DJScias/Experiencer2#13] [Ideas]
Hello again @DJScias,

My passion for this addon has resulted in me learning a smidgen of lua. Thus, I developed an extremely basic addon which hides the default status tracking bars.

I did very brief testing, including turning on and off the reputation/honor bars through the Blizzard UI and changing reputations/honor/experience bars through Experiencer. I have yet to encounter an issue visually, or through BugSack or BugGrabber.

Therefore, I'm left with two options:
1) I publish it on Curseforge as a standalone addon and mention it being inspired by Experiencer 2.0 and being a simpler solution. (Instead of having to download heavier and more complex addons like Bartender, Dominos, or Edit Mode Expanded).
2) You take the code, and having more skill than me, implement in into Experiencer if possible. If this was to happen, I expect nothing in return - think of it as appreciation for your work!

 I still have no idea how addon development works, or even github... so I welcome your input. :)

I'm not even sure how to upload the addon to GitHub LOL! But I'm kind of proud of it, it only seems to use 2KB constantly (5KB upon load). 
![Screenshot 2024-09-09 230124](https://github.com/user-attachments/assets/be046cfe-ce78-4b43-bfd1-8becb98fe4e0)

**Comment by ghost** [comment]
I finally figured out how to upload it to GitHub :')

https://github.com/yaybab/HideStatusBars

**Comment by DJScias** [comment]
Hey there @JBabbb.

I think this is super awesome and cool that you did this.

However regardless of whether or not I'll add it (not sure yet, have other stuff to do at the moment)..
You should self-publicize it definitely as well. Because as you said there might be people who want **just that** without other unnecessary parts of an addon attached.

Also that way you can start earning rewards on curseforge/wago/etc.. as well because if it turns out to be a hit then you'd potentially lose out on your hard-earned rewards.

I would say that that would be the best of two worlds: You self-release it and gain the fruits of your labor and if I do add it in some shape or way you'll get accredited as well.

**Comment by ghost** [comment]
Thank you so much @DJScias, that makes a lot of sense.

I'm sorry for opening it as an issue, I didn't even know you could make alternative conversations here on github!

All the best :)

**Reply by DJScias** [reply]
> Thank you so much @DJScias, that makes a lot of sense.
> 
> I'm sorry for opening it as an issue, I didn't even know you could make alternative conversations here on github!
> 
> All the best :)

Oh no, no worries at all about the issue creation.
I hadn't opened this "Discussions" part on github yet so there wasn't any means for you to write it here.

Your question basically made me think it would be great to open it up as that's probably the better route for this. :)
